README.md

```diff
@@ -56,12 +63,14 @@ Note: `version.scala.binary` refers to major releases of scala ie. 2.11, 2.12 or
 
 make sure you have set a version in your scalafmt.conf 
 ```yaml
-version = "1.5.1"
+version = "2.5.2"
 ```
 
 [licenseimg]: https://img.shields.io/badge/Licence-Apache%202.0-blue.svg
 [licenselink]: ./LICENSE
 
+[BuildImg]: https://github.com/SimonJPegg/mvn_scalafmt/workflows/Release/badge.svg
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[definition-case] Do not use uppercase characters in definition labels](https://app.codacy.com/manual/Antipathy_org/mvn_scalafmt/pullRequest?prid=5511915)